src/main/webapp/app/entities/event-activity/event-activity-update.tsx

```diff
@@ -126,31 +105,26 @@ export class EventActivityUpdate extends React.Component<IEventActivityUpdatePro
                   </Label>
                   <AvField id="event-activity-durationInDay" type="text" name="durationInDay" />
                 </AvGroup>
-                <AvGroup>
-                  <Label id="nameLabel" for="event-activity-name">
-                    <Translate contentKey="clubmanagementApp.eventActivity.name">Name</Translate>
-                  </Label>
-                  <AvField id="event-activity-name" type="text" name="name" />
-                </AvGroup>
                 <AvGroup>
                   <Label id="descriptionLabel" for="event-activity-description">
                     <Translate contentKey="clubmanagementApp.eventActivity.description">Description</Translate>
                   </Label>
                   <AvInput id="event-activity-description" type="textarea" name="description" />
                 </AvGroup>
-                <Button tag={Link} id="cancel-save" to="/entity/event-activity" replace color="info">
-                  <FontAwesomeIcon icon="arrow-left" />
-                  &nbsp;
-                  <span className="d-none d-md-inline">
+                <span className="text-error">{errorMessage ? errorMessage.response.data.detail : ''}</span>
```

Comment:
is this error message from startDate AvInput? it is not displayed when there is error

Reply:
this is error message when get error from BE
the startDate Input will show below the input